src/turnkeyml/files_api.py

```diff
@@ -370,7 +328,7 @@ def benchmark_files(
                     name=onnx_name,
                     script_name=onnx_name,
                     file=file_path_absolute,
-                    build_model=not build_only,
+                    build_model=True,
```

Comment:
Could we get rid of the `build_model` arg now that benchmark is a stage?
Same applies to `ModelInfo`

Reply:
Yeah there's a lot of these little remnants hanging around. I'm taking care of a bunch in the discover stage PR, but honestly we'll need a full codebase review to see what's left hanging around too.